src/cpu/platform.hpp

```diff
@@ -43,6 +44,14 @@
 #endif
 #endif // defined(DNNL_X64) + ... == 0
 
+// Using Arm Compute Library kernels is optional for AArch64 builds
+// and can be enabled with the DNNL_AARCH64_USE_ACL CMake option
+#if defined(DNNL_AARCH64) && defined(USE_ACL)
+#define _AARCH64_ACL 1
+#else
+#define _AARCH64_ACL 0
+#endif
```

Comment:
Probably not worth having an extra macro, as it is only used for `DNNL_AARCH64_ACL_ONLY(...)`.

Suggest having something like:
```cpp
#define DNNL_X64_ONLY(...) Z_CONDITIONAL_DO(DNNL_X64, __VA_ARGS__)
#define DNNL_AARCH64_ONLY(...) Z_CONDITIONAL_DO(DNNL_AARCH64_ONLY, __VA_ARGS__)
#define DNNL_PPC64_ONLY(...) Z_CONDITIONAL_DO(DNNL_PPC64_ONLY, __VA_ARGS__)

// Using Arm Compute Library kernels is optional for AArch64 builds
// and can be enabled with the DNNL_AARCH64_USE_ACL CMake option
#if defined(DNNL_AARCH64) && defined(USE_ACL)
#define DNNL_AARCH64_ACL_ONLY(...) __VA_ARGS__
#else
#define DNNL_AARCH64_ACL_ONLY(...)
#endif
```